src/applicant.js

```diff
@@ -0,0 +1,21 @@
+//import our utility object for getting data
+import applicantApi from './applicant-api.js';
+
+//reference all the DOM elements we need to update
+const name = document.getElementById('name');
+const meowTime = document.getElementById('meow-time');
+const die = document.getElementById('die');
+const lives = document.getElementById('lives');
+
+// get the applicant name
+// const searchParams = new URLSearchParams(window.location.search);
```

Comment:
This functionality needs to be here so you can grab an applicant of the URL to be able to pass it to the detail page on line 15 with ```applicantApi.get(id);